modules/nodes-pods-autoscaling-custom-about.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-autoscaling-custom.adoc
+
+:_content-type: CONCEPT
+[id="nodes-pods-autoscaling-custom-about_{context}"]
+= Understanding the custom metrics autoscaler
+
+The custom metrics autoscaler uses the Kubernetes-based Event Driven Autoscaler (KEDA) and is built on top of the {product-title} horizontal pod autoscaler (HPA).
+
+The Custom Metrics Autoscaler Operator scales your pods up and down based on custom, external metrics from specific applications. Your other applications continue to use other scaling methods. You can configure a list of predefined link:https://keda.sh/#scalers[scalers], also known as triggers, which are the source of events and metrics that the custom metrics autoscaler uses to determine how to scale. The custom metrics autoscaler uses a metrics API to convert the external metrics to a form that {product-title} can use. The custom metrics autoscaler creates a horizontal pod autoscaler (HPA) that performs the actual scaling.
```

Comment:
Is Red Hat supporting all of the Scalers listed by the upstream? IE: if a customer finds a bug with one of these scalers (or the upstream adds a scaler in the middle of a 4.Y version's lifecycle) can a customer take advantage of it? Can they get patches and security updates for the scalers? 

If the answer is no; to this then we shouldn't like to the upstream here; but instead we should list what Red Hat is supporting. 

All in all I don't have an issue linking to the upstream here; but we have to clarify this paragraph with a support clause / call out that said (more or less): Red Hat provides and only supports the ability to use scalers; the source / support for each individual scaler is offered by the provider of at scaler (please see the provider for support options), etc, etc.

Reply:
We will not support all scaler but only one "prometheus" . We should be make it absolutely clear that we will not support any upstream scaler but only prometheus

Reply:
@gauravsingh85 We explicitly document the Prometheus, Kafka, CPU, and Memory triggers.  Do we want to remove all but Prometheus?

Reply:
yup only prometheus .. and leave a note that more is coming ( something like that)

Reply:
@gauravsingh85 The internal Prometheus Cluster Monitoring and external Proetheus source? Or just internal?

Reply:
CPU and Memory scaler is just a wrapper around existing HPA funcitonality, there's no harm in supporting these. But we can keep it to the next release.